From: Michael Tokarev <mjt@tls.msk.ru>
Subject: skip unpacking of edk2 blobs
Date: Sat, 01 Apr 2023 18:46:55 +0300
Debian-Specific: yes
Forwarded: not-needed

Debian strips the blobs from qemu.orig.tar.gz so the code
fails to unpack the missing files.

diff --git a/pc-bios/meson.build b/pc-bios/meson.build
index a7224ef469..ddaedd95af 100644
--- a/pc-bios/meson.build
+++ b/pc-bios/meson.build
@@ -1,4 +1,4 @@
 roms = []
-if unpack_edk2_blobs
+if unpack_edk2_blobs and false
   fds = [
     'edk2-aarch64-code.fd',
